discovery_engine_core/web-api2/src/error/common.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use actix_web::http::StatusCode;
+use displaydoc::Display;
+use serde::Serialize;
+use thiserror::Error;
+use tracing::error;
+
+use crate::impl_application_error;
+
+use super::application::ApplicationError;
+
+/// The requested document was not found.
+#[derive(Debug, Error, Display, Serialize)]
+pub struct DocumentNotFound;
+impl_application_error!(DocumentNotFound => NOT_FOUND);
+
+/// The requested property was not found
```

Comment:
:art: similarly for other docstrings below
```suggestion
/// The requested property was not found.
```

Reply:
Adding a `.` is in conflict with error formatting using `Display` generated by `display_doc`.